scripts/customMessage.coffee

```diff
@@ -0,0 +1,36 @@
+# Description:
+#   work from home scripts
+#
+# Dependencies:
+#   None
+#
+# Configuration:
+#   None
+#
+# Commands:
+#   @caari post in <room_no> message: <message>
+#
+# Author:
+#   Sarah Noor
+
+
+#perosnalTestRoom = process.env.PER_TEST_CHANNEL
+
+
+module.exports = (robot) ->
+  robot.hear /post in (.*)/i,(msg)->
+  	if msg.message.room == 'G284H7WEQ' #custom_message channel id
```

Comment:
how about we add user check here? 
like if username == waleed || user == sarah.noor
than proceed?